.github/workflows/release.yaml

```diff
@@ -13,8 +13,9 @@ jobs:
       - uses: actions/checkout@v3
       - name: Install conda
         run:  .github/scripts/install-conda.sh
-      - shell: bash
-        name: Format Check
+      - name: Docs Link Check
+        run: .github/scripts/link-check.sh
```

Comment:
Run the link checker as part of the `release` workflow (run only for `develop` -> `main` merges).